docs/agent-api.asciidoc

```diff
@@ -157,10 +157,27 @@ Arguments:
 
 * `type` - The type of the transaction (string). Defaults to `custom`
 
+* `options` - Options to modify the created transaction (object). Defaults to `undefined`
```

Comment:
can you also explicitly specify both `managed` and `reuse` as list of options here before we explain about managed transactions.